git/foundations_git/introduction_to_git.md

```diff
@@ -47,7 +47,7 @@ This section contains questions for you to check your understanding of this less
 *   <a class="knowledge-check-link" href="#text-editor-and-git">What are the differences between Git and a text editor in terms of what they save and their record keeping?</a>
 *   <a class="knowledge-check-link" href="#git-local">Does Git work at a local or remote level?</a>
 *   <a class="knowledge-check-link" href="#github-remote">Does GitHub work at a local or remote level?</a>
-*   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=8oRjP8yj2Wo">Why is Git useful for an individual developer?</a>
+*   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=D-HVcKon2ak">Why is Git useful for developers?</a>
```

Comment:
Looks like we forgot to update this knowledge link in this PR: https://github.com/TheOdinProject/curriculum/pull/25257

The knowledge links correspond to lesson content and assignments, we'll need to use the video from the assignment here:

```suggestion
*   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=2ReR1YJrNOM">Why is Git useful for developers?</a>
```

Reply:
Okay, thanks for your support @KevinMulhern.